src/styles/variables.css

```
@@ -0,0 +1,11 @@
+:root {
+  --cardBorderColor: #e5e5e5;
```

stick to one naming style
```suggestion
  --card-border-color: #e5e5e5;
```